ts/components/session/SessionIcon.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export enum SessionIconTypes {
+  Arrow = 'arrow',
+  ChatBubble = 'chatBubble',
+  Check = 'check',
+  Chevron = 'chevron',
+  CirclePlus = 'circlePLus',
+  Contacts = 'contacts',
+  Ellipses = 'ellipses',
+  Emoji = 'emoji',
+  Eye = 'eye',
+  File = 'file',
+  Gear = 'gear',
+  Globe = 'globe',
+  Microphone = 'microphone',
+  Moon = 'moon',
+  Reply = 'reply',
+  Search = 'search',
+  Send = 'send',
+  Star = 'star',
+}
+
+const icons = {
+  Arrow: {
```

Comment:
so you would need to update all those fields to their `SessionIconTypes` equivalent: 
```
'arrow':  {
path:
      'M33.187,12.438 L6.097,12.438 L16.113,2.608 C16.704,2.027 16.713,1.078 16.133,0.486 C15.551,-0.105 14.602,-0.113 14.011,0.466 L1.407,12.836 C1.121,13.117 0.959,13.5 0.957981241,13.9 C0.956,14.3 1.114,14.685 1.397,14.968 L14.022,27.593 C14.315,27.886 14.699,28.032 15.083,28.032 C15.466,28.032 15.85,27.886 16.143,27.593 C16.729,27.007 16.729,26.057 16.143,25.472 L6.109,15.438 L33.187,15.438 C34.015,15.438 34.687,14.766 34.687,13.938 C34.687,13.109 34.015,12.438 33.187,12.438',
    viewBox: '0 -4 37 37',
  },
```

Reply:
What do you mean by this?

Reply:
i think he means what i said above:
```
Yep it would be possible if you set the keys in icons to SessionIconTypes.<iconType>
E.g Arrow -> SessionIconTypes.Arrow.
```